app/javascript/Search/__tests__/Search.test.jsx

```diff
@@ -17,28 +17,42 @@ describe('<Search />', () => {
   });
 
   it('should have no a11y violations', async () => {
-    const { container } = render(<Search />);
+    const props = {
```

Comment:
Test updates because `searchTerm` and `setSearchTerm` props come from a parent component.